backend/uclapi/roombookings/management/commands/trigger_webhooks.py

```diff
@@ -121,18 +130,23 @@ def webhook_filter(booking):
 
             webhooks_to_enact[idx]["payload"] = payload
 
-            if payload["content"] != {}:
+            if payload["content"] != {} and webhook["url"] != "":
                 unsent_requests.append(
-                    grequests.post(
-                        webhook["url"], json=payload, headers={
+                    session.post(
+                        webhook["url"], json=payload,
+                        headers={
                             "User-Agent": "uclapi-bot/1"
                         }
                     )
                 )
         self.stdout.write(
             "Triggering {} webhooks.".format(len(unsent_requests))
         )
-        grequests.map(unsent_requests)
+        if("debug" in options):
```

Comment:
Where is this set?

Reply:
its a flag you can pass to the command --debug, when running individually instead of via the gencache. It just enables easier debugging in the future if we notice this takes a long time.

Reply:
fantastic, thanks!